pkg/runtime/reconciler.go

```diff
@@ -155,6 +155,11 @@ func (r *resourceReconciler) Reconcile(req ctrlrt.Request) (ctrlrt.Result, error
 		"account", acctID,
 		"role", roleARN,
 		"region", region,
+		// All the fields for a resource that do not change during reconciliation
+		// can be initialized during resourceLogger creation
```

Comment:
does this comment also apply to the three value pairs above it?

Reply:
Yes, but they are constant but at controller level. 
I made the comment for fields we were deriving from the resource every time earlier